Doc/c-api/time.rst

```diff
@@ -0,0 +1,132 @@
+.. highlight:: c
+
+PyTime C API
+============
+
+.. versionadded:: 3.13
+
+PyTime API
+----------
+
+The PyTime_t API is written to use timestamp and timeout values stored in
+various formats and to read clocks.
+
+The :c:type:`PyTime_t` type is an integer to support directly common arithmetic
+operations such as ``t1 + t2``.
+
+The PyTime_t API supports a resolution of ``1`` nanosecond. The
+:c:type:`PyTime_t` type is signed to support negative timestamps. The supported
+range is around [-292.3 years; +292.3 years]. Using the Unix epoch (January
+1st, 1970), the supported date range is around [1677-09-21; 2262-04-11].
+
+Time formats:
+
+* Seconds.
+* Seconds as a floating pointer number (C :c:expr:`double`).
+* Milliseconds (10\ :sup:`-3` seconds).
+* Microseconds (10\ :sup:`-6` seconds).
+* 100 nanoseconds (10\ :sup:`-7` seconds), used on Windows.
+* Nanoseconds (10\ :sup:`-9` seconds).
+* :c:expr:`timeval` structure, 1 microsecond (10\ :sup:`-6` seconds).
+* :c:expr:`timespec` structure, 1 nanosecond (10\ :sup:`-9` seconds).
+
+Integer overflows are detected and raise :exc:`OverflowError`. Conversion to a
```

Comment:
This paragraph and the next don't make much sense given the four functions that are part of the public API. None of them raise `OverflowError` and the rounding modes aren't relevant.